### ISSUE TITLE
Privacy Notification

### ISSUE THREAD

**Issue by JSpragueDesign** [IITIDIDX597/sp_2021_team1#16]
**Project Goal:** S Lab is a tailored integrative learning and collaboration platform for clinicians that combines the latest research and tacit knowledge gained from experience in a practical way, while at the same time foster deeper learning experiences in order to deliver better AbilityLab Patient care.

**Hill Statement:** Individual Clinicians can reference relevant, continuously evolving information for their patient's therapy needs to self-manage their approach & patient care plan development in a single platform.

**Sub-Hill Statements:** 

 1. Clinicians will be able to view the most popular articles or research areas and areas that need further focus for the lab.

### **Story Details:**
As a: Clinician
I want: to be able to see the notification that my search history is visible to the administrator
So that : I am aware of the Lab's transparency and privacy policies

**Comment by akallish** [comment]
@JSpragueDesign this story is not clear to me based on the subhill statement.

As a: Clinician
I want: to be able to see the notification that my search history is visible to the administrator (why is this important as an enabling task to a clinician?)
So that : I am aware of the Lab's transparency and privacy policies (what about this is important to a clinician?)

**Comment by JSpragueDesign** [comment]
@akallish This came from a conversation about if administrators will be able to track the use of the system, Clinicians should be made aware that is happening.

Perhaps that is a sub-hill itself? Is it appropriate to add more sub-hills at this point?

**Comment by akallish** [comment]
> @akallish This came from a conversation about if administrators will be able to track the use of the system, Clinicians should be made aware that is happening.
> 
> 
> 
> Perhaps that is a sub-hill itself? Is it appropriate to add more sub-hills at this point?
> 
> 

@JSpragueDesign first address the existing sub-hills. If there is a tealizathat in writing a stories that there are missing details to the sub-hills, sure write additional ones.

**Comment by JSpragueDesign** [comment]
@sliu131 @kanalchhajed @zixuan-zhaozhao 
Perhaps this story is out of scope for the Subhills we used to define this Epic.

I propose we Icebox it, allowing us to come back if ever needed.

**Comment by zixuan-zhaozhao** [comment]
Thanks, @JSpragueDesign I agree to keep it icebox, we can come back and change it if we needed it in the future.

**Comment by kanalchhajed** [comment]
Agree! I apologize, I moved it to the backlog by mistake. Will move it back to icebox if we all agree to it! @sliu131 @JSpragueDesign @zixuan-zhaozhao
